Classes/Hooks/DefaultUploadFolder.php

```diff
@@ -54,7 +56,7 @@ public function getDefaultUploadFolder($params, BackendUserAuthentication $backe
                     $subFolder
                 );
             } catch (FolderDoesNotExistException $e) {
-                // todo: try to create the folder
+                $uploadFolder = $this->createUploadFolder($subFolder);
```

Comment:
This only creates folders when they're formatted like `1:myUploadFolder`, not when it's formatted like `myUploadFolder` (without `1:` in front). Is this correct? According to the documentation the latter is also a correct value (see https://github.com/beechit/default_upload_folder/blob/master/README.md?plain=1#L17)